vita3k/shaders-builtin/vulkan/compile.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+for %%i in (*.vert *.frag) do "C:\VulkanSDK\Bin\glslangValidator.exe" -V "%%~i" -o "%%~i.spv"
```

Comment:
new line missing

Reply:
I think this can be turned into CMake using `add_custom_command`, etc.
Also check `Vulkan::glslangValidator` https://cmake.org/cmake/help/latest/module/FindVulkan.html

Reply:
This was just a personal helper to help me compile glsl files into spv. The shaders are really simple and should rarely (if ever) be changed so I will remove this file.

Moreover because Vita3K is now linking against Vulkan dynamically (and not statically), the Vulkan SDK is no longer required to compiler Vita3K, even with Vulkan enabled (there is no `find_package(Vulkan REQUIRED)` in the cmake anymore). It would be annoying to make it a requirement only to compile 2 static shaders.